### PR TITLE
Update README.md (Bug Number)

### PR DESCRIPTION
**Pull request by gradalajage**

### CODE REVIEW

**Comment by gradalajage** [comment]
Hi this a comment

**Comment by kyleestes** [comment]
Hi who are you?

File: .github/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [![Build Status](https://dev.azure.com/3M-Bluebird/AzurePlatform/_apis/build/status/serenity/build/test-pr?branchName=master)](https://dev.azure.com/3M-Bluebird/AzurePlatform/_build/latest?definitionId=198&branchName=master)
 
+
```
Comment (kyleestes) [inline .github/README.md:3]: Why??

Comment (gradalajage) [reply .github/README.md:3]: Just because of testing

**Comment by kyleestes** [comment]
FYI, the user @gradalajage  is me, Kyle Estes, my personal account -- I'm just testing/exploring the flow of sumitting a PR from a fork of this repo

**Pull request closed without merging**